sheets/vorkurs.cls

```diff
@@ -94,11 +123,14 @@
   \ifthenelse{\value{excerciseCounter}=1}{}{
   \vspace{2em}
   }
-  \large\textbf{Aufgabe \arabic{excerciseCounter}:} #1\normalsize\\
+  \section{#1}
+  %\large\textbf{Aufgabe \arabic{excerciseCounter}:} #1\normalsize\\
   \medskip
 }
-\newcounter{subexcerciseCounter}
-\newcommand{\subexcercise}{\stepcounter{subexcerciseCounter}\alph{subexcerciseCounter}}
```

Comment:
This change breaks compilation of sheets 1 and 4?
Is this intended?
Should the sheets be change to use `\section` and `\subsection` ?

Reply:
I just found out they should be replaced by enumerations and sheet4 has not been fixed yet. Sheet 1 isn't actually broken.